web/themes/custom/avengers/js/welcome-message.js

```diff
@@ -0,0 +1,22 @@
+(function ($, Drupal, drupalSettings) {
+  Drupal.behaviors.welcomeMessage = {
+    attach: function (context, settings) {
+      if (settings.welcomeMessage) {
+        var message = $('<div class="welcome-message"></div>').text(settings.welcomeMessage);
+
+        message.css({
+          'background-color': '#f0f8ff',
```

Comment:
Why have we added styling through js? styling through js should be avoided. Styling should be done in scss file only incase of any dynamic styling is required only that could be done through js. But even in case of dynamic styling add a class dynamically and styling through scss only.